cli/state.rs

```diff
@@ -52,6 +53,8 @@ pub struct State {
   pub argv: Vec<String>,
   pub permissions: DenoPermissions,
   pub flags: flags::DenoFlags,
+  pub config: Option<Vec<u8>>,
+  pub config_file_name: Option<String>,
```

Comment:
Is there any reason not to combine these two in a tuple I.E. `pub config_file: Option<(String, Vec<u8>)>`?

Reply:
Not everything will access the file name, in particular, when hashing for caching, it will only access the config.